openedx/features/course_card/admin.py

```diff
@@ -31,25 +44,39 @@ class Meta:
 
 
 class CardModelAdmin(admin.ModelAdmin):
+    """
+    To Integrate the Card form with Django Admin.
+    """
     form = CardModel
 
 
 class CourseRerunStateModel(ModelForm):
-    def __init__(self, *args, **kwargs):
-
-        super(CourseRerunStateModel, self).__init__(*args, **kwargs)
+    """
+    Model form to create/update CourseRerun which contains source_course_key  field.
+    source_course_key: (string) id of the course.
+    """
 
     class Meta:
         model = CourseRerunState
         fields = ['source_course_key']
 
 
 class CourseRerunStateModelAdmin(admin.ModelAdmin):
-
+    """
+    To Integrate the CourseRerunState form with Django Admin.
+    """
     def has_delete_permission(self, request, obj=None):
+        """
+        Disabling the delete permission for CourseRerunState
+        return: False
+        """
         return False
 
     def get_actions(self, request):
+        """
+        Disabling the delete_selected action for CourseRerunState
+        return: actions
+        """
```

Comment:
```
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value
    """
```

Please try to stick to the edx recommended docstring style, here and elsewhere. It's not enforced by the linters, but recommended, and internally agreed upon.